src/Screens/Homepage/Homepage.css

```diff
@@ -0,0 +1,51 @@
+@import url('https://fonts.googleapis.com/css2?family=Dancing+Script:wght@600&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Langar&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Langar&display=swap');
+
+.home-background{
+  background-image: url();
+  background-size: 100%;
+}
+
+.home-title{
+  color: green;
+  font-family: 'Dancing Script', cursive;
+  font-size: 6em;
+  margin-top: 0px;
+  margin-bottom: 0px;
+}
+
+.home-tagline{
+  color: black;
+  font-family: 'Langar', cursive;
+  font-size: 15px;
+  margin-left: 50px;
+}
+
+.homestead{
+  width: 750px;
+  height: 250px;
```

Comment:
This photo is out of proportion; at 750px wide, it should be 457px high. Also, the image is too far compressed; since these are moderately small images, the compression should not be higher than 80% quality.